### ISSUE TITLE
mobile browser support

### ISSUE THREAD

**Issue by bclabs-kylian** [MatthewLarner/cross-domain-storage#46]
Hello.

I am trying this library and it seems like it does not work with mobile browsers.
I tested with Safari and Chrome with my iphone xs and ipad. It cannot read the storage.

Can you please check this?

**Comment by bclabs-daniel** [comment]
Same issue!

**Comment by Andresdamelio** [comment]
Have you found a solution for this?

**Comment by bclabs-kylian** [comment]
@Andresdamelio  no. decided not to use this solution. switching to other work around.
